js/authFunctions.js

```diff
@@ -0,0 +1,114 @@
+// authFunctions.js
+
+import {
+  noneEmailCheck,
+  nonePasswordCheck,
+  failEmailCheck,
+  failPasswordCheck,
+  noneNameCheck,
+  samePasswordCheck,
+  errorMessage,
+} from "./sign.js";
+
+export function checkemail(email, emailerror) {
+  if (noneEmailCheck(email.value) === false) {
```

Comment:
이런 표현이 더 일반적이라 제안드려요
```suggestion
  if (!noneEmailCheck(email.value)) {
```